src/docker-outside-of-docker/install.sh

```diff
@@ -99,6 +99,22 @@ find_version_from_git_tags() {
     echo "${variable_name}=${!variable_name}"
```

Comment:
Can we bump patch version of the Feature?

Reply:
bumped the patch version